keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -228,6 +233,216 @@ def generate_image(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def text_to_image_ptp(
```

Comment:
Curious what @LukeWood thinks about this, but I think we should consider other names for this method, like:
`prompt_to_prompt` or `text_to_image_prompt_to_prompt`

Reply:
(I think more generally we should probably remove the `ptp` abbreviation and replace it with the full `prompt_to_prompt`, e.g in the naming of the utils file)

Reply:
Fixed and updated in the PR comment!